_src/static/sass/components/_copy.scss

```diff
@@ -0,0 +1,118 @@
+// ------------------------------
+// edX Pattern Library: Components - Copy
+
+// About: Contains base styling for copy (general text and type used in content and functionaly).
+
+// #GLOBAL
+// #INDIVIDUAL COPY CASES
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.copy {
+    @extend %reset-copy;
```

Comment:
I'm a little worried about the explicit resets for each of these patterns. We don't want to end up with font family, color, and size/line height set on every element a la LMS spans. Can we let the cascade assume that responsibility and have the reset available when the cascade goes awry?